internal/characters/tighnari/skill.go

```diff
@@ -0,0 +1,68 @@
+package tighnari
+
+import (
+	"github.com/genshinsim/gcsim/internal/frames"
+	"github.com/genshinsim/gcsim/pkg/core/action"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+)
+
+var skillFrames []int
+
+const (
+	skillHitmark           = 20
```

Comment:
replace skillHitmark = 20 with skillRelease = 15 and add a travel param that defaults to 5